src/js/components/userProfilePanel/connectedUserProfilePanel.js

```diff
@@ -3,21 +3,18 @@
 import {connect} from 'react-redux';
 import UserProfilePanel from './userProfilePanel';
 import { logout } from '../../actions/authentication';
-import jwtdecode from 'jwt-decode';
+import { requestUpdateUserInfo } from '../../actions/userInfo';
+// import jwtdecode from 'jwt-decode';
 
 const ConnectedUserProfilePanel = connect(
-    state => {
-        if(!state.authentication.isAuthenticated) {
-            return {};
-        }
-        let decodedJWT = jwtdecode(state.authentication.jwt);
-        return {
-            name: decodedJWT.user_name,
-            orcid: decodedJWT.user_orcid_id
-        };
-    },
+    state => ({
+        name: state.userInfo.user_name,
+        orcid: state.userInfo.user_orcid_id,
+        email: state.userInfo.user_email
```

Comment:
@Laludia maybe eventually but for now this meets the acceptance criteria for the story.